08/SUMMARY.md

```diff
@@ -1,2 +1,2 @@
-Matrix is a very powerful tool for manipulating vectors. By mastering how to use matrices, you can freely translate, scale and rotate shapes. Since the technique can be equally applied to anything expressed by vectors, we will look at many more advanced use of matrices later in this book.
```

Comment:
A better fix would be "**A m**atrix is" keeping the subject of a matrix as a tool in the singular. Matrices are referred to later in the paragraph tying the two words together. If you did keep "Matrices are" you should change "tool" to "tools" as well.

Reply:
cool. Seams the change here is directly erasing the paragraph